src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <threadsafe>false</threadsafe>
+  <sessions-enabled>true</sessions-enabled>
```

Comment:
Suggestion: Maybe we should disable the sessions? I don't think we'll need it on our project.

Reply:
We can probably disable them for now, and if we will need them in the future, we can turn them back on